apps/api-journeys/tsconfig.app.json

```
@@ -3,8 +3,10 @@
   "compilerOptions": {
     "outDir": "../../dist/out-tsc",
     "module": "commonjs",
-    "types": ["node", "express"]
+    "types": ["node"],
+    "emitDecoratorMetadata": true,
+    "target": "es2015"
   },
-  "exclude": ["**/*.spec.ts", "tests/*.ts"],
+  "exclude": ["**/*.spec.ts", "**/*.test.ts"],
```

want to standardise on spec.ts

```suggestion
  "exclude": ["**/*.spec.ts"],
```